Not found page

Dumitrescu-Dragos opened the pull request.

Dumitrescu-Dragos committed:
Refactor App.tsx and add not-found.png

Dumitrescu-Dragos committed:
Refactor App.tsx and add not-found.png

Dumitrescu-Dragos committed:
Solve conflicts

Dumitrescu-Dragos committed:
Solve conflicts

Dumitrescu-Dragos committed:
Solve conflicts

Dumitrescu-Dragos committed:
Solve conflicts

Dumitrescu-Dragos committed:
Add NotFoundComponent

Dumitrescu-Dragos committed:
Add NotFoundComponent

Dumitrescu-Dragos committed:
Fix App.tsx

Dumitrescu-Dragos committed:
Fix App.tsx

andrianwritek commented on VikingQuiz.UI/src/Components/NotFoundComponent/NotFoundComponent.tsx:
```diff
@@ -1,12 +1,17 @@
 import './NotFoundComponent.css';
 
-const redirectComponent: any = (props: any) => {
+const notFoundComponent: any = (props: any) => {
```
component with uppercase

andrianwritek commented on VikingQuiz.UI/src/Components/NotFoundComponent/NotFoundComponent.tsx:
```diff
@@ -1,12 +1,17 @@
 
-export default redirectComponent;
\ No newline at end of file
+export default notFoundComponent;
```
component with uppercase

andrianwritek commented on VikingQuiz.UI/src/Components/NotFoundComponent/NotFoundComponent.tsx:
```diff
@@ -1,12 +1,17 @@
+          <h1 className="very-large">grrr...</h1>
+          <p className="large">we're angry too that this doesn't work</p>
+          <p className="small">now let's get back to serious business'</p>
+          <footer><BottomLogo /></footer>
```
why footer and logo is there, it must be in master render?

vasi-axinte commented on VikingQuiz.Api/Utilities/UserExtensions.cs:
```diff
@@ -1,16 +1,16 @@
+        public static int GetUserId(this IEnumerable<Claim> user)
         {
-            var strId = user.Claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
+            var strId = user.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
```
Try to use first letter of the entity name in lambda expressions (i.e. u instead of x)

vasi-axinte commented on VikingQuiz.Api/Utilities/UserExtensions.cs:
```diff
@@ -1,16 +1,16 @@
         {
-            var strId = user.Claims.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
+            var strId = user.FirstOrDefault(x => x.Type == ClaimTypes.NameIdentifier).Value;
             return int.Parse(strId);
```
Please use full name for variables, do not abreviate them

Dumitrescu-Dragos committed:
Refactor UserExtensions.cs

Dumitrescu-Dragos committed:
Refactor UserExtensions.cs

Dumitrescu-Dragos committed:
Refactor NotFoundComponent, FormInput and App Components:
- Move footer to App.tsx
- Refactor NotFoundComponent (capitalize name and delete foooter)
- Refactor FormInput (capitalize name)
- Remove footer from existing components (Login, Signup)

Dumitrescu-Dragos committed:
Refactor NotFoundComponent, FormInput and App Components:
- Move footer to App.tsx
- Refactor NotFoundComponent (capitalize name and delete foooter)
- Refactor FormInput (capitalize name)
- Remove footer from existing components (Login, Signup)

vasi-axinte commented:
Everything is cool, but solve the conflict first. After that, we can merge this branch into develop

Dumitrescu-Dragos committed:
Merge branch 'notFoundPage' of https://github.com/AvaSpa/VikinQuiz into notFoundPage

# Conflicts:
#	VikingQuiz.UI/src/Components/FormComponent/FormInput/FormInput.tsx
